src/MAUI/Maui.Samples/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml.cs

```diff
@@ -94,37 +95,31 @@ private async Task Initialize()
             }
         }
 
-        private void ButtonClick(object sender, EventArgs e)
+        private void StyleChooser_SelectedIndexChanged(object sender, EventArgs e)
         {
-            _ = StyleChooser();
+            // Get the style name and index of the selected item.
+            _ = ChangeStyle(StyleChooser.SelectedIndex, StyleChooser.SelectedItem.ToString());
         }
 
-        private async Task StyleChooser()
+        private async Task ChangeStyle(int styleIndex, string styleName)
```

Comment:
I would rename this to ChangeStyleAsync.  This is a [general recommendation](https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/async-scenarios#important-info-and-advice) by the .NET team:
> Add "Async" as the suffix of every async method name you write.

Reply:
Good to know!